march_hardware_interface/config/test_setup/controllers.yaml

```diff
@@ -9,7 +9,14 @@ march:
     pdb_state:
       type: march_pdb_state_controller/MarchPdbStateController
       publish_rate: 50
-    trajectory:
+    trajectory_position:
```

Comment:
Why trajectory_position here instead of just trajectory? For the march4 this is still trajectory

Reply:
I get that there is another trajectory_effort part, but why is it different from the march4 controllers.yaml?

Reply:
The march4 controllers have to cooperate with the gait schedular, which expects the name trajectory controller. For the test joint it doesn't matter, because the gait schedular is not used. Therefore it is even easier if both controllers are defined, since then only the name of the actuationmode have to be changed in the yaml.

Reply:
For uniformity I decided that I'll still change it to look like the march4.yaml.